src/components/orgs/orgPermissions.vue

```diff
@@ -70,7 +76,9 @@ export default {
 
     async fetchPermissions($state) {
        try {
+        this.loading=true;
         const response = await this.getMembers({ uuid: this.org.uuid, page: this.page });
+        this.loading=false;
```

Comment:
This set to false, in this case, is unnecessary due it has already an `this.loading = false` in the finally block